test/regression/solvers/delta_eddington.hpp

```diff
@@ -0,0 +1,46 @@
+// Copyright (C) 2023-2024 National Center for Atmospheric Research
+// SPDX-License-Identifier: Apache-2.0
+//
+// Tests for tuvx::DeltaEddington.
+#pragma once
+
+#include <tuvx/radiative_transfer/solvers/delta_eddington.hpp>
+
+#ifdef __cplusplus
+extern "C" {
```

Comment:
why do we need `extern` here?

Reply:
It tells the compiler that these functions will be called by c code, which in our case is actually fortran code using `iso_c_binding`. It prevents the name mangling that would happen if these were just C++ functions.

Reply:
Ah, I see. That makes sense. Why are we using Fortran test driver here?

Reply:
It's because all the code that parses the configuration files and prepares the input data for the solver is in Fortran.

Reply:
Thanks for the clarification.